test/extended/etcd/etcd_backup_noconfig.go

```diff
@@ -0,0 +1,120 @@
+package etcd
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	configv1alpha1 "github.com/openshift/api/config/v1alpha1"
+	exutil "github.com/openshift/origin/test/extended/util"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kube-openapi/pkg/util/sets"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	"github.com/pkg/errors"
+)
+
+const (
+	defaultBackupCRName       = "default"
+	testSchedule              = "*/5 * * * *"
+	testTimeZone              = "UTC"
+	testRetentionType         = configv1alpha1.RetentionTypeNumber
+	testRetentionNumber       = 3
+	backupServerContainerName = "etcd-backup-server"
+)
+
+var _ = g.Describe("[sig-etcd][OCPFeatureGate:EtcdBackupNoConfig][Serial] etcd", func() {
+	defer g.GinkgoRecover()
+	oc := exutil.NewCLIWithoutNamespace("etcd-backup-no-config").AsAdmin()
+
+	// The following test executes an e2e test for automated etcd backup no-config.
+	// It starts by applying the a config.openshift.io/v1alpha1/Backup CR.
+	// It waits for a new revision of static pods to be deployed.
+	// It verifies that backups are being taken according to the specified schedule and retention policy.
+	g.It("is able to apply the no-config backup configuration [Timeout:30m][apigroup:config.openshift.io/v1alpha1]", func(ctx context.Context) {
+
+		g.GinkgoT().Log("applying Backup CR")
+		backupCR := createDefaultBackupCR(testSchedule, testTimeZone, testRetentionNumber)
+		_, err := oc.AdminConfigClient().ConfigV1alpha1().Backups().Create(context.Background(), backupCR, metav1.CreateOptions{})
+		o.Expect(err).ToNot(o.HaveOccurred())
+
+		g.GinkgoT().Log("waiting for api servers to stabilize on the same revision")
+		err = waitForEtcdToStabilizeOnTheSameRevision(g.GinkgoT(), oc)
+		err = errors.Wrap(err, "timed out waiting for etcd pods to stabilize on the same revision")
+		o.Expect(err).ToNot(o.HaveOccurred())
+
+		g.GinkgoT().Log("ensuring all etcd pods have etcd-backup-server container enabled")
+		err = ensureBackupServerContainerEnabled(ctx, oc)
+		o.Expect(err).ToNot(o.HaveOccurred())
+	})
+})
+
+func ensureBackupServerContainerEnabled(ctx context.Context, oc *exutil.CLI) error {
+	etcdPods, err := oc.AdminKubeClient().CoreV1().Pods("openshift-etcd").List(ctx, metav1.ListOptions{})
+	if err != nil {
+		err = errors.Wrap(err, "failed to list etcd pods")
+		return err
+	}
+	for _, etcd := range etcdPods.Items {
+		for _, c := range etcd.Spec.Containers {
+			if c.Name != backupServerContainerName {
+				continue
+			}
+
+			if err := ensureBackupServerArgs(c); err != nil {
+				return fmt.Errorf("etcd-backup-server within etcd-pod %v has incorrect configurations %v", etcd.Name, err)
+			}
+		}
+	}
+	return nil
```

Comment:
that should be an error, no?

Reply:
huh, why ?! 

I fail fast above, if we reached here so no error 

please correct me if I am wrong

Reply:
yes, but this function will not error if the pods does not have the sidecar container.

Reply:
good catch 👍🏽

Reply:
used a counter to track it, thx tons 👍🏽

Reply:
ptal, I updated 👍🏽

Reply:
looks good! Thanks